### PR TITLE
added translations - default virtual learning link

### PR DESCRIPTION
**Pull request by dartajax**
As the title states - I added translations for default virtual learning link - this will be used for small groups so they can have a pre-set value for this very important new field.

refs #5579

### CODE REVIEW

File: translations/en-us.yaml [hunk 1]
```diff
@@ -93,6 +93,7 @@ general:
   dashboard: Dashboard
   deactivate: Deactivate
   defaultInstructors: "Default Instructors"
+  defaultVirtualLearningLink: "Default Virtual Learning Link"
```
Comment (stopfstedt) [inline translations/en-us.yaml:96]: `defaultVirtualLearningLink` should be below `defaultLocation`.

File: translations/es.yaml [hunk 2]
```diff
@@ -93,6 +93,7 @@ general:
   dashboard: Tablero
   deactivate: Desactivar
   defaultInstructors: "Instructores Predeterminados"
+  defaultVirtualLearningLink: "Enlace de Aprendizaje Virtual Predeterminado"
```
Comment (stopfstedt) [inline translations/es.yaml:96]: see comment above.

**Review by stopfstedt — CHANGES_REQUESTED** [review]

File: translations/es.yaml (hunk 2 above)
Comment (dartajax) [reply translations/es.yaml:96]: oh oops - I changed it from a different title and then it got out of alphabetization - I will fix this.

**Review by stopfstedt — APPROVED** [review]
LGTM

**Pull request merged**